### ISSUE TITLE
Fix nuget push warning for license

### ISSUE THREAD

**Issue by biohazard999** [xenial-io/Tasty#45]
When publishing a new version via tags there is a warning:

```txt
warn : All published packages should have license information specified. Learn more: https://aka.ms/deprecateLicenseUrl.
  Created https://www.nuget.org/api/v2/package/ 607ms
```

**Comment by Paramethod** [comment]
🤔 Odd. There is no <licenseUrl> tag in the nuspec. Maybe because there is a <requireLicenseAcceptance> and no <license>?

**Comment by biohazard999** [comment]
I didn't investigate yet. But i am still puzzeled why the Tool didnt get published either 🤔

**Comment by Paramethod** [comment]
Perhaps it will fix itself when you fix #42 and add the `<license>` details in the nuspec.

**Comment by biohazard999** [comment]
Fixed with #42
